infra/modules/azure_function_app/locals.tf

```diff
@@ -1,29 +1,31 @@
 locals {
-  location_short = var.environment.location == "italynorth" ? "itn" : var.environment.location == "westeurope" ? "weu" : var.environment.location == "germanywestcentral" ? "gwc" : "neu"
-  project        = "${var.environment.prefix}-${var.environment.env_short}-${local.location_short}"
-  domain         = var.environment.domain == null ? "-" : "-${var.environment.domain}-"
+  location_short  = var.environment.location == "italynorth" ? "itn" : var.environment.location == "westeurope" ? "weu" : var.environment.location == "germanywestcentral" ? "gwc" : "neu"
```

Comment:
can't we use a unique map shared by all modules instead of replicating this logic in every module?

```hcl
variable "environment" {
  type = object({
    location = string
  })
}

locals {
  location_map = {
    "italynorth"        = "itn"
    "westeurope"        = "weu"
    "germanywestcentral" = "gwc"
    # Default (se la location non è presente nella mappa)
    "default"           = "neu"
  }
}

location_short = lookup(local.location_map, var.environment.location, local.location_map["default"])

```